src/main/java/com/google/sps/servlets/FavouriteServlet.java

```diff
@@ -113,4 +113,4 @@ private Recommendation convertEntityToRecommendation(Entity recommendation) {
     int crowdRating = (int) (long) recommendation.getProperty("crowd-rating");
     return new Recommendation(id, name, category, lat, lng, description, costRating, crowdRating);
   }
-}
\ No newline at end of file
+}
```

Comment:
I'm not really sure what you changed here?

Reply:
if you are referring to the last line, it's just adding enter after the end of file